### ISSUE TITLE
MultiSelect: Accessibility Issues with the component

### ISSUE THREAD

**Issue by ramtejsudani6** [primefaces/primeng#12161]
### Describe the bug

When using MultiSelect component, we are running accessibility insights on the webpage and observed multiple failures that requires fix from the primeng component to add the accessibility attributes. 

1. aria-required-children
<img width="545" alt="image" src="https://user-images.githubusercontent.com/83031111/200905242-126cb446-4eaa-4556-bd37-55cb2f31c5a0.png">
2. label 
<img width="541" alt="image" src="https://user-images.githubusercontent.com/83031111/200905511-21e6ee75-a4a9-4a69-b06a-d191e698cf27.png">
3. listitem-
<img width="556" alt="image" src="https://user-images.githubusercontent.com/83031111/200905659-6340f81f-93d3-4a40-876d-aa8698d9064c.png">

The expected output is the multiselect component passes all the accessibility rules. 

### Environment

Production

### Angular version

13.3.1

### PrimeNG version

13.3.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

v14.18.3

### Steps to reproduce the behavior

We have verified the issue on below Primeng URL
URL: (https://www.primefaces.org/primeng/multiselect)

Steps to reproduce:

Expand the multiselect dropdown and run Accessibility insights for web. 

### Expected behavior

For Multiselect component, all the accessibility insights rules have to pass.

**Comment by ramtejsudani6** [comment]
This issue still exists in latest release
